src/Config/apiConfig.ts

```diff
@@ -0,0 +1,5 @@
+export const baseUrl = "https://api.nasa.gov/mars-photos/api/v1/rovers";
+
+export const apiKey = "qj9vwpFaaunBJQeJYnDtZjW1tW7ckM3gBZdQUh28";
+
+export const dedaultSol = "1000";
```

Comment:
By convention, the constant identifiers are in uppercase (eg `BASE_URL` ) - no need to change, though